### ISSUE TITLE
A11y_language-server-protocol_AI4W: Ensure links are distinguished from surrounding text in a way that does not rely on color (a)

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/debug-adapter-protocol#374]
[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Github Tags:** #A11yMAS;#A11ySev2;#A11yTCS;#AI4W;#Benchmark;#ChromiumEdge;#DesktopWeb;#FTP;#WCAG 1.4.1#UseOfColor;#Win10;#link-in-text-block;#P2_languageserverprotocol_Web_Sep2021;#Visual Studio Code Client;

## Environment Details:
Application Name: Language Server Protocol Specification - 3.15 Page
#URL: https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#textDocument_hover
Windows Version: Win10 22H2 19045.2251
Chromium Edge version: Version 110.0.1587.49 (Official Build) (64-bit)
## Repro Steps:

1. **Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix**
2. Launch the url: https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#textDocument_hover
3. Run accessibility insights for web and see the results.
## Issue: 
Ensure links are distinguished from surrounding text in a way that does not rely on color (link-in-text-block - https://dequeuniversity.com/rules/axe/4.6/link-in-text-block?application=msftAI)

Target application: Specification - https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#textDocument_hover

Element path: p:nth-child(1) > a

Snippet: <a href="https://github.com/Microsoft/vscode-languageserver-node">here</a>
### Similar issue is observed in scenario:-

1. https://microsoft.github.io/language-server-protocol/specifications/specification-3-16/
2. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/
3. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.18/specification/
4. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_workspaceFolders
5. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#lsp
6. https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#hover
7. https://microsoft.github.io/language-server-protocol/specifications/lsif/0.5.0/specification/
8. https://microsoft.github.io/language-server-protocol/specifications/lsif/0.6.0/specification/
9. https://microsoft.github.io/language-server-protocol/implementors/servers/
10. https://microsoft.github.io/language-server-protocol/implementors/tools/
11. https://microsoft.github.io/language-server-protocol/overviews/lsif/overview/
12. https://microsoft.github.io/language-server-protocol/overviews/lsp/overview/
13. https://microsoft.github.io/language-server-protocol/specifications/lsif/0.4.0/specification/
14. https://microsoft.github.io/language-server-protocol/specifications/lsif/0.5.0/specification/
15. https://microsoft.github.io/language-server-protocol/specifications/lsif/0.6.0/specification/

## How to fix: 
Fix any of the following:
  The link has insufficient color contrast of 2.28:1 with the surrounding text. (Minimum contrast is 3:1, link text: #216fc9, surrounding text: #373a3c)
  The link has no styling (such as underline) to distinguish it from the surrounding text

Environment: Microsoft Edge version 110.0.1587.49

====

This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
## User Impact:
If there is no visual under lining to the links even without hovering mouse, Monochromatic users will not be able to distinguish between normal text and a link so they could miss the mouse hovering to get the under lining, over the actual links considering them as a plain text.
## Attachement:

![374_A11y_language-server-protocol](https://user-images.githubusercontent.com/93374043/220113636-2c4a35bc-1347-4cc2-855e-6875eed02fe7.png)
![374_A11y_language-server-protocol_without ai4w](https://user-images.githubusercontent.com/93374043/220113683-dc3e1693-9c07-412a-b5b4-21f2b439cfce.png)

**Comment by Yashi1201** [comment]
Verified the issue in env https://microsoft.github.io/language-server-protocol/specifications/specification-3-15/#textDocument_hover
Main issue is not repro
![image](https://user-images.githubusercontent.com/93374043/228428713-3976534c-266d-4802-882f-5319fbd13be5.png)

All Similar issues are also not repro
![image](https://user-images.githubusercontent.com/93374043/228434339-ba85e8c2-8ed8-4e80-94a3-0ed2aaa4016b.png)

![image](https://user-images.githubusercontent.com/93374043/228434513-ad023014-31f9-485b-b8b1-5e2594d2b2a8.png)

![image](https://user-images.githubusercontent.com/93374043/228434609-62ae17bc-2fb7-4927-9edf-d9fe53722b5b.png)

Hence, closing the bug.
